feature.h

```diff
@@ -53,132 +53,132 @@
 
 #define FEATURE_FC_IS_ENABLED \
     ( \
-	(CURRENT_FEATURE_BUNDLE >= FEATURE_BUNDLE_515 && \
```

Comment:
```
   !!!!!!!   DO NOT EDIT THIS FILE   !!!!!!!
   This file is built by regen/feature.pl.
   Any changes made here will be lost!
```

Reply:
The PR also updates that file.

Reply:
Oh, I see that these probably get fixed by the heredoc fixes in regen/feature; that's probably worth calling out as a separate commit, since it is effectively a code change.